gramps_webapi/api/resources/filters.py

```diff
@@ -184,6 +184,23 @@ class CustomFilterSchema(FilterSchema):
     )
 
 
+class FiltersResources(ProtectedResource, GrampsJSONEncoder):
+    """Filters resources."""
+
+    @use_args(
+        {},
+        location="query",
+    )
```

Comment:
Just curious, what is the purpose of this empty schema?

Reply:
Yeah I was unsure if I should add that, have done so in a few places but not everywhere it could be.  It is a way of having webargs validate the user didn't provide any query parms and if they did it treats it as an error.